pkg/test/framework/components/istio/istio.go

```diff
@@ -59,6 +62,43 @@ func Setup(i *Instance, cfn SetupConfigFn) resource.SetupFn {
 	}
 }
 
+// SetupWithContext is a setup function that will deploy Istio
+// with the provided config and context functions on Kubernetes environment.
+func SetupWithContext(i *Instance, cfn SetupConfigFn, ctxFn SetupContextFn) resource.SetupFn {
```

Comment:
If we are going to add this can you reimplement Setup() to be `SetupWithContext(i, cfg, nil)`?

Reply:
Removed SetupWithContext() and made Setup(i *Instance, cfn SetupConfigFn, ctxFns ...SetupFunctions).